intermediate_notebooks/examples/linear_regression_demo.ipynb

```diff
@@ -752,7 +758,7 @@
cGXq9MqqjGWqneYz654yDkgPlKdEZuPZQWajzcGhjju9bDw7yGy0OTjUcaeXjRcBmo02B4c67vSy8ewgs9Hm4FCnDJ3eMNQI8iJAs9Hm2Up18k6J7PbMpjLOAmrGs4PMRpeDQ4p2nV41IMxs2oyo3OwHutORtxoQd0dsZv3itFJOtVNd4ZnAUNXpzCYPiJtZGTg45JR2Zl+vk47cA+JmVgYODjll6fg76cjLMCBuZubgkFO7jr/TjtyzgMysDDwgndPZJy7aYTYRMDcoPd2lOkyeBWRmg+bgkJOrf5rZOHBwKMBn9mY26jzmYGZmDRwczMysgYODmZk1cHAwM7MGDg5mZtbAwcHMzBp4KmsO3S7PbWZWVg4OGQ3TfRbMzDrltFJGre6zYGY2ahwcMvJ9FsxsnDg4ZOT7LJjZOHFwyMj3WTCzceIB6YxcjdXMxomDQw6uxmpm48JpJTMza+DgYGZmDRwczMysgYODmZk1cHAwM7MGDg5mZtbAU1lHiKvGmlm3ODiMCFeNNbNuclppRLhqrJl1UymDg6RXS7pH0r9KWjHo9gwDV401s24qXXCQNAF8AngNcBBwuqSDBtuq8nPVWDPrptIFB+BlwL9GxI8iYgvwD8DrBtym0nPVWDPrpjIGh2ngxzWPf5Js24GkMyStkbRm48aNfWtcWS1bPM3K5YcyPX8KAdPzp1i5/FAPRptZIWWcraSUbdGwIeKTwCcBlixZ0vD8OHLVWDPrljJeOfwE2L/m8X7AQwNqi5nZWCpjcPg+8CJJB0raGfgd4NoBt8nMbKyULq0UEVsl/TfgemAC+HREbBhws8zMxkrpggNARHwN+Nqg22FmNq7KmFYyM7MBU8TwT/SRtBF4YNDtaON5wE8H3YgS8nFp5GOSzsclXSfH5YCI2CvtiZEIDsNA0pqIWDLodpSNj0sjH5N0Pi7penVcnFYyM7MGDg5mZtbAwaF/PjnoBpSUj0sjH5N0Pi7penJcPOZgZmYNfOVgZmYNHBzMzKyBg0MPSPq0pEcl/aBm256SbpB0b/J9j0G2sd8k7S/pm5LukrRB0nuS7eN+XHaR9D1JtyfH5YJk+1gfF6jc+EvSOklfTR77mEj3S7pT0npJa5JtPTkuDg698ffAq+u2rQBujIgXATcmj8fJVuCsiHgJcAzw7uQOf+N+XH4FHBcRhwNHAK+WdAw+LgDvAe6qeexjUvGKiDiiZm1DT46Lg0MPRMS3gMfqNr8OuDz5+XJgWV8bNWAR8XBE/Evy8xNU/tNP4+MSEfFk8nAy+QrG/LhI2g84Gfjbms1jfUxa6MlxcXDon+dHxMNQ6SiBvQfcnoGRtBBYDHwXH5dq+mQ98ChwQ0T4uMBHgD8BttdsG/djApUTh29IWivpjGRbT45LKauy2uiS9BzgKuC9EfELKe3Gf+MlIrYBR0iaD1wj6ZBBt2mQJL0WeDQi1ko6dtDtKZmlEfGQpL2BGyTd3asd+cqhfx6RtA9A8v3RAben7yRNUgkMn4+Iq5PNY39cqiJiE3AzlfGqcT4uS4HflnQ/8A/AcZI+x3gfEwAi4qHk+6PANcDL6NFxcXDon2uBtyU/vw34ygDb0neqXCJ8CrgrIj5c89S4H5e9kisGJE0BrwTuZoyPS0ScExH7RcRCKneCvCki3sIYHxMASc+WtFv1Z+BVwA/o0XHxCukekPRF4FgqpXQfAc4DVgNXAguAB4E3RUT9oPXIkvRy4NvAnTyTR/5TKuMO43xcDqMyiDhB5WTtyoi4UNKvMcbHpSpJK/1xRLx23I+JpN+gcrUAlSGBL0TERb06Lg4OZmbWwGklMzNr4OBgZmYNHBzMzKyBg4OZmTVwcDAzswYODmZm1sDBwUaGpNdLCkkvbvO6+ZLe1eG+nmyyfVtSTvkHkr4kadcmr/tOJ/vPStJLJN0naV7yeJ6kb0h6az/2b8PLwcFGyenALVRW1bYyH+goOLSwOSmnfAiwBXhn7ZOqmBcR/7FH+99BRNxFZcX1a5NNfwncExGf6cf+bXg5ONhISAr6LQXeQU1wkPRWSXckN9P5bLJ5FfCC5Az/YkkL627M9MeSzk9+Xp1UwNxQUwUzq28DL0ze/y5JlwH/Auxfe+XRpI1IektyI6D1kv4mqd76bEnXJa/9gaTTMrTjUuAPJb0hOUbvy/k5bAy5KquNimXA1yPi/0l6TNKRVG6k82dUKln+VNKeyWtXAIdExBEwV0K8mbdHxGNJ3aPvS7oqIn7WrjGSdgJeA3w92bQI+L2IeFfyfPV1B6e1UdJLgNOS7bNJYHkz8BTwUEScnLxu9+T714DfrxZmqxUR35B0CbAS+K2ImG3XfjNfOdioOJ1KBU+S76cDxwFfjoifAhSsN/NHkm4HbgP2B17U5vVTyb0Z1lCpc/OpZPsDEXFbyuubtfF44KVUAtL65PFvUKlN9UpJH5L0nyLi58nvnZQWGGp8B/hwte4/gKQPtPksNsZ85WBDLyk8dhxwiKSgUsQugEuS7+1sZccTpV2S9z2WSpXU34yIpyXdXH2uhc3VK5Ka9kHljD+1+U3aKODyiDin4QnppcBJwEpJ34iIC9u0CeAg4O9q3uPX8f9/a8FXDjYK3gh8JiIOiIiFEbE/cB+wHjg1CR7UpJWeAHar+f1HgL0l/ZqkZ/HM4O3uwONJYHgxlXtfd9uNTdp4I/DG5KYu1ZvIHyBpX+DpiPgc8D+BIzPu52Aq5Z2rFlM5PmapHBxsFJzOM6WMq66iMjB9EfB/k9TQhwGSMYNbkwHdi5Mc/IVUyod/lcrsHqiMF+wk6Q7gA1RSS10VERuatPGHwLlUbgl5B3ADsA9wKPC9JNX0Z8AHoTLmkASOBpL2BzbV3Ksa4AgcHKwFl+w2G0OSPgX8QURsb/tiG0sODmZm1sBpJTMza+DgYGZmDRwczMysgYODmZk1cHAwM7MGDg5mZtbAwcHMzBo4OJiZWQMHBzMza/D/AZzReTbKSJIOAAAAAElFTkSuQmCC\n",
```

Comment:
Are you sure that 17th character should be a U?

Reply:
It's an image (plot), I think it should be right. 🤔